server.html

```diff
@@ -32,6 +32,7 @@ <h1 id="server-side">I want to add CORS support to my server</h1>
 			<li><a href="server_tomcat.html">Tomcat</a></li>
 			<li><a href="server_virtuoso.html">Virtuoso</a></li>
 			<li><a href="server_wcf.html">WCF</a></li>
+			<li><a href="server_bfe.html">BFE</a></li>
```

Comment:
Instead of creating a separate page that needs to be independently maintained, what about linking out to the actual documentation page? That will make this easier to keep up-to-date in the future. Thanks!

```suggestion
			<li><a href="https://github.com/bfenetworks/bfe/blob/develop/docs/en_us/modules/mod_cors/mod_cors.md">BFE</a></li>
```

Reply:
> Instead of creating a separate page that needs to be independently maintained, what about linking out to the actual documentation page? That will make this easier to keep up-to-date in the future. Thanks!

It has been revised according to your suggestion，Thanks!